### PR TITLE
kertish-dos: renamed the formula

### PR DESCRIPTION
**Pull request by freakmaxi**
- [ x ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ x ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ x ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ x ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ x ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ x ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by freakmaxi** [comment]
Hello,

According to the instruction in the following URL [https://docs.brew.sh/Rename-A-Formula](https://docs.brew.sh/Rename-A-Formula) `brew audit --strict` should pass. It passes, but here, it considers that it is a new formula and there shouldn't be `  bottle do` section in the formula. 

Should I remove that section and retry or Should I use `brew bump-formula-pr`?

File: Formula/k/kertish-dos.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
-class KertishDfs < Formula
-  desc "Kertish FileSystem and Cluster Administration CLI"
-  homepage "https://github.com/freakmaxi/kertish-dfs"
-  url "https://github.com/freakmaxi/kertish-dfs/archive/refs/tags/v22.2.0147.tar.gz"
+class KertishDos < Formula
+  desc "Kertish Object Storage and Cluster Administration CLI"
+  homepage "https://github.com/freakmaxi/kertish-dos"
```
Comment (chenrui333) [inline Formula/k/kertish-dos.rb:3]: can you cut a new release per the project rename?

Comment (freakmaxi) [reply Formula/k/kertish-dos.rb:3]: There will not be anything new soon (at least for 3 months) other than the renaming. Is it a kind of blocker for this PR for homebrew?

Comment (chenrui333) [reply Formula/k/kertish-dos.rb:3]: nope, but it would be nice, otherwise, we dont need to regenerate the artifacts.

Comment (chenrui333) [reply Formula/k/kertish-dos.rb:3]: actually, we still need to generate the bottle per the spec change.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @freakmax!

**Pull request merged**